daemons/fenced/cts-fence-helper.c

```diff
@@ -93,6 +82,23 @@ static const int st_opts = st_opt_sync_call;
 static int expected_notifications = 0;
 static int verbose = 0;
 
+static void
+mainloop_test_done(const char *origin, bool pass)
+{
+    if (pass) {
+        crm_info("SUCCESS - %s", origin);
+        mainloop_iter++;
+        mainloop_set_trigger(trig);
+        result.execution_status = PCMK_EXEC_UNKNOWN;
```

Comment:
why not PCMK_EXEC_DONE?

Reply:
I don't remember, that would make more sense. It doesn't matter b/c cts-fence-helper only compares it against a couple of failure values, so I'll change that in a follow-up too